docs/tutorials/dataset_creation/observation_space_subseting.py

```diff
@@ -0,0 +1,108 @@
+# fmt: off
+"""
+Observation space subseting
```

Comment:
I would change it to "Collecting a subset of dictionary space keys with StepDataCallback"

Reply:
Done